charts/argo-cd/templates/argocd-application-controller/poddisruptionbudget.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if .Values.controller.pdb.enabled }}
-apiVersion: policy/v1beta1
+apiVersion: {{ include "argo-cd.podDisruptionBudget.apiVersion" . }}
```

Comment:
I think this change is harmless :eyes:
https://github.com/argoproj/argo-helm/pull/1288/files#r878700532